app/helpers/application_helper.rb

```diff
@@ -13,8 +13,20 @@ def divide_in_chunks(arr, chunk_size)
     chunks
   end
 
-  def get_img_url(video)
-    video.img ? "/tmp/#{video.img}" : 'http://dummyimage.com/665x375/000000/00a2ff'
+  def get_large_cover_url(video)
```

Comment:
My apologies, I assumed your db column was `large_cover_url`. Your method should be working, have you used pry to debug `large_cover.url`?